### PR TITLE
Make version list work after typing @ for library id

### PR DESCRIPTION
**Pull request by YangLyu89**

### CODE REVIEW

File: src/LibraryManager.Vsix/Json/Completion/CompletionController.cs [hunk 1]
```diff
@@ -151,10 +151,10 @@ private void OnCommitted(object sender, EventArgs e)
             }
         }
 
-        private JSONParseItem GetItemBeforePosition(int pos, JSONEditorDocument document)
+        private JSONParseItem GetItemBeforePosition(int pos, JSONComplexItem document)
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Json/Completion/CompletionController.cs]: >document [](start = 77, length = 8)

This is no longer taking a document, so it should be renamed.

File: src/LibraryManager.Vsix/Json/Completion/CompletionController.cs [hunk 2]
```diff
@@ -173,7 +173,7 @@ private JSONParseItem GetItemBeforePosition(int pos, JSONEditorDocument document
                     if (item is JSONComplexItem)
                     {
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Json/Completion/CompletionController.cs]: use `if (item is JSONComplexItem complexItem)` here so you don't have to cast below.

File: src/LibraryManager.Vsix/Json/Completion/CompletionController.cs (hunk 1 above)
Comment (YangLyu89) [reply src/LibraryManager.Vsix/Json/Completion/CompletionController.cs]: Will rename

File: src/LibraryManager.Vsix/Json/Completion/CompletionController.cs (hunk 2 above)
Comment (YangLyu89) [reply src/LibraryManager.Vsix/Json/Completion/CompletionController.cs]: Will do

**Comment by YangLyu89** [comment]
https://github.com/aspnet/LibraryManager/issues/135 is fixed in this one

**Pull request merged**